components/gantry/singleaxis/singleaxis_test.go

```diff
@@ -31,8 +31,9 @@ var fakeFrame = &referenceframe.LinkConfig{
 var badFrame = &referenceframe.LinkConfig{}
 
 var (
-	count     = 0
-	pinValues = []int{1, 1, 0}
+	count            = 0
+	defaultPinValues = []int{1, 1, 0}
+	limitPinValues   = []int{1, 0, 1, 0, 1, 1, 0}
```

Comment:
I don't really understand what the difference between these two are / why these test changes are needed for this code change, could you give me some more context/add comments to this file?

Reply:
thank you! maybe rename back to `pinValues` since theres now only one. Also comment is still unclear since they are not all alternating?

Reply:
they can be fully alternating, it doesn't super matter it's just that the first couple need to be alternating for the homing tests. basically the issue before was they were never getting set low. at some point it needs to be set low. changed it to fully alternating tho because that is clearer

Reply:
thinking to leave it as default because if they need to be changed for some reason in adding future tests, more versions could be added

Reply:
maybe just add "during homing" to the comment for future clarity? otherwise everything lgtm!